### DISCUSSION TITLE
this->param? Shouldn't it be this->parm?

### DISCUSSION THREAD

**Discussion by rica-carv** [e107inc/e107#5279] [Q&A Core]
Shouldn't the `this->param` call be `this-parm` here in [news_shortcode](https://github.com/e107inc/e107/blob/baa171cfb8de93947c6cfb7e6477cf558f94fce1/e107_core/shortcodes/batch/news_shortcodes.php#L66C1-L69C4) file?
```
		if(!empty($this->param['titleLimit']))
		{
			$parm['limit'] = $this->param['titleLimit'];
		}
```

**Comment by tgtje** [comment]
not sure however it is used more.
i see it as parm=function      where param=argument
